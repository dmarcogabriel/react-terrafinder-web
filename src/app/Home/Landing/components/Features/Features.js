import React from 'react';
import { useHistory } from 'react-router-dom';
import { Box, Card, Typography, Button } from '@mui/material';
import { ArrowForward } from '@mui/icons-material';
import { CARDS_CONTENT } from './cardsContent.object';
import { FEATURES } from './features.string';
import {
  FeaturesContainer,
  FeaturesBackgroundImage,
  FeaturesBoxFlex,
  FeaturesTitle,
  FeaturesSubtitle,
} from './styles';

export const Features = () => {
  const history = useHistory();

  const handleCreateProperty = () => history.push('/create-property?step=1');
  const handleFindProperty = () => history.push('/search-property');

  return (
    <FeaturesContainer sx={{ py: 4 }}>
      <FeaturesBackgroundImage />

      <FeaturesTitle>{FEATURES.title}</FeaturesTitle>

      <FeaturesBoxFlex sx={{ mx: { xs: 2, md: 5 }, my: 2 }}>
        <Box sx={{ ml: 2, mr: 2 }}>
          <FeaturesSubtitle sx={{ my: 3 }}>
            {FEATURES.subtitle}
          </FeaturesSubtitle>
          {FEATURES.descriptions.map((description, i) => (
            <Typography key={String(i)} sx={{ my: 3 }}>
              {description}
            </Typography>
          ))}
          <FeaturesBoxFlex sx={{ alignItems: 'center', my: { xs: 3, md: 0 } }}>
            <Button
              sx={{ mr: { xs: 0, md: 1 }, my: { xs: 1 } }}
              data-testid="createProperty"
              variant="contained"
              color="primary"
              endIcon={<ArrowForward />}
              onClick={handleCreateProperty}
            >
              {FEATURES.buttons.create}
            </Button>
            <Button
              sx={{ ml: { xs: 0, md: 1 }, my: { xs: 1 } }}
              data-testid="findProperty"
              variant="contained"
              color="success"
              endIcon={<ArrowForward />}
              onClick={handleFindProperty}
            >
              {FEATURES.buttons.find}
            </Button>
          </FeaturesBoxFlex>
        </Box>

        <Box
          sx={{
            px: { xs: 3 },
            display: { md: 'grid' },
            gridTemplateColumns: { md: 'repeat(2, 1fr)' },
          }}
        >
          {CARDS_CONTENT.map((card) => (
            <Card
              key={card.key}
              sx={{
                m: { xs: 2, md: 1 },
                p: 2,
                background: card.colors.bg,
                color: card.colors.text,
                minHeight: '150px',
              }}
            >
              <Box
                sx={{
                  display: 'flex',
                  flexDirection: 'row',
                  alignItems: 'center',
                  mb: 1,
                }}
              >
                <card.IconComponent sx={{ mr: 2, color: card.colors.icon }} />
                <Typography
                  sx={{ fontWeight: 'bold', lineHeight: '28px' }}
                  variant="body1"
                >
                  {card.title}
                </Typography>
              </Box>
              <Typography variant="body2" sx={{ lineHeight: '28px' }}>
                {card.text}
              </Typography>
            </Card>
          ))}
        </Box>
      </FeaturesBoxFlex>
    </FeaturesContainer>
  );
};
