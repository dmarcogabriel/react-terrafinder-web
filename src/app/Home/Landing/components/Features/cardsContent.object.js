import { AccessTime, TrackChanges, SupportAgent } from '@mui/icons-material';

export const CARDS_CONTENT = [
  {
    key: 'c1',
    IconComponent: AccessTime,
    colors: {
      icon: '#eee',
      bg: '#62ac43',
      text: '#eee',
    },
    title: 'Pague apenas pelo tempo do seu anúncio',
    text:
      'Para anunciar na plataforma, selecione o plano de anúncio que melhor se encaixa para você e pague apenas pelo tempo em que ele ficar publicado.',
  },
  {
    key: 'c2',
    colors: {
      icon: '#62ac43',
      bg: '#eee',
      text: '#000',
    },
    IconComponent: TrackChanges,
    title: 'Quero um anúncio profissional',
    text:
      'Nós oferecemos um serviço que profissionaliza o seu anúncio. Temos uma equipe que pode ir até seu imóvel, fotografa-lo e gravar vídeos sensacionais com drones. Impossível não vender assim!',
  },
  {
    key: 'c3',
    colors: {
      icon: '#62ac43',
      bg: '#eee',
      text: '#000',
    },
    IconComponent: SupportAgent,
    title: 'Precisa de ajuda?',
    text:
      'Entre em contato através dos nossos canais de atendimento.\n\nWhatsApp: (14)99999-8888',
  },
];
