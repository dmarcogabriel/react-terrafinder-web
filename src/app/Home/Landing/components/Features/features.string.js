export const FEATURES = {
  title: 'Funcionalidades e Benefícios do Terrafinder',
  subtitle: 'Conheça e aprenda a utilizar o Terrafinder',
  descriptions: [
    'Você que quer vender seu imóvel rural, encontrou o lugar ideal para anuncia-lo. É simples, rápido e eficiente.',
    ' Ou caso tenha interesse em comprar algum imóvel rural em determinado lugar, você achou o lugar correto! Escolha uma das opções abaixo para entender e aprender a criar ou procurar sua propriedade rural.',
  ],
  buttons: {
    create: 'Quero anunciar meu imóvel',
    find: 'Quero achar uma propriedade',
  },
};
