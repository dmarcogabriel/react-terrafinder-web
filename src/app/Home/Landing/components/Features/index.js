import React from 'react';
// import Button from 'common/components/atm/Button';
// import { MdArrowForward } from 'react-icons/md';
import { useHistory } from 'react-router-dom';
import { Box, Card, Typography, Button } from '@mui/material';
import { ArrowForward } from '@mui/icons-material';
import classes from './Features.module.scss';
import { CARDS_CONTENT } from './cardsContent.object';

export default function Features() {
  const history = useHistory();

  const handleCreateProperty = () => history.push('/create/property?step=1');
  const handleFindProperty = () => history.push('home/search-property');

  return (
    <div className={classes.container}>
      <div className={classes.backgroundImage} />

      <h1 className={classes.title}>
        Funcionalidades e Benefícios do Terrafinder
      </h1>

      <div className={classes.content}>
        <div className={classes.column}>
          <h2 className={classes.subtitle}>
            Conheça e aprenda a utilizar o Terrafinder
          </h2>

          <p className={classes.text}>
            Você que quer vender seu imóvel rural, encontrou o lugar ideal para
            anuncia-lo. É simples, rápido e eficiente.
          </p>

          <p className={classes.text}>
            Ou caso tenha interesse em comprar algum imóvel rural em determinado
            lugar, você achou o lugar correto! Escolha uma das opções abaixo
            para entender e aprender a criar ou procurar sua propriedade rural.
          </p>

          <div className={classes.actionButtons}>
            <Button
              data-testid="createProperty"
              variant="contained"
              color="primary"
              endIcon={<ArrowForward />}
              onClick={handleCreateProperty}
            >
              Quero anunciar meu imóvel
            </Button>
            <Button
              data-testid="findProperty"
              variant="contained"
              color="success"
              endIcon={<ArrowForward />}
              onClick={handleFindProperty}
            >
              Quero achar uma propriedade
            </Button>
          </div>
        </div>

        <Box
          sx={{
            px: { xs: 3 },
            display: { md: 'grid' },
            gridTemplateColumns: { md: 'repeat(2, 1fr)' },
          }}
        >
          {CARDS_CONTENT.map((card) => (
            <Card
              key={card.key}
              sx={{
                m: 1,
                p: 1,
                background: card.colors.bg,
                color: card.colors.text,
              }}
            >
              <Box
                sx={{
                  display: 'flex',
                  flexDirection: 'row',
                  alignItems: 'center',
                  mb: 1,
                }}
              >
                <card.IconComponent sx={{ mr: 2, color: card.colors.icon }} />
                <Typography sx={{ fontWeight: 'bold' }} variant="body1">
                  {card.title}
                </Typography>
              </Box>
              <Typography variant="body2">{card.text}</Typography>
            </Card>
          ))}
        </Box>
      </div>
    </div>
  );
}
