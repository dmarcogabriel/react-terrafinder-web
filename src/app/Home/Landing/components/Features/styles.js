import React from 'react';
import { styled } from '@mui/material/styles';
import { Box, Typography } from '@mui/material';
import backgroundImage from './featuresBackground.png';

export const FeaturesBackgroundImage = styled(Box)({
  position: 'absolute',
  height: '100%',
  width: '100%',
  opacity: 0.1,
  top: 0,
  bottom: 0,
  left: 0,
  right: 0,
  backgroundImage: `url(${backgroundImage})`,
  zIndex: -1,
});

export const FeaturesTitle = styled((props) => (
  <Typography variant="h4" {...props} />
))({
  textAlign: 'center',
  fontWeight: 'bolder',
});

export const FeaturesSubtitle = styled((props) => (
  <Typography variant="h5" {...props} />
))({
  textAlign: 'center',
});

export const FeaturesBoxFlex = styled(Box)(({ theme }) => ({
  display: 'flex',
  flexDirection: 'column',
  [theme.breakpoints.up('md')]: {
    flexDirection: 'row',
  },
}));
