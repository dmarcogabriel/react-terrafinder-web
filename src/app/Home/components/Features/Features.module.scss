@import 'styles';

.container {
  position: relative;
  background-color: #f0e3d0;
  padding: 5rem 0;
  z-index: 5;

  .backgroundImage {
    position: absolute;
    height: 100%;
    width: 100%;
    opacity: 0.1;
    top: 0;
    bottom: 0;
    left: 0;
    right: 0;
    z-index: -1;
  }

  .title {
    text-align: center;
    font-weight: bold;
    font-size: 2rem;
  }

  .content {
    padding: 2.5rem 1rem;
    display: flex;
    flex-direction: column;
    justify-content: center;

    @media ($desktop) {
      padding: 2.5rem 5rem;
      flex-direction: row;
    }

    .column {
      margin: 0 2rem;
    }

    .subtitle {
      font-weight: normal;
      font-size: 2rem;
      margin: 2rem 0;
    }

    .text {
      font-size: $size-md;
      margin: 1.5rem 0;
    }

    .actionButtons {
      display: flex;
      flex-direction: column;
      align-items: center;
      justify-content: space-between;
      margin-top: 4rem;

      @media ($desktop) {
        flex-direction: row;
      }
    }

    .actionButton {
      margin: 0.75rem 0;
      display: flex;
      justify-content: space-between;
      align-items: center;

      @media ($desktop) {
        margin: 0;
      }
    }

    .cards {
      background: black;
      display: flex;
      align-items: center;
      justify-content: center;

      .card {
        border-radius: 0.15rem;
        background: $white;
        color: $black;
        box-shadow: $shadow-base;

        .cardTitle {
          font-weight: bold;
          font-size: 1rem;
        }

        .cardText {
          font-size: 0.8rem;
        }
      }
    }
  }
}
