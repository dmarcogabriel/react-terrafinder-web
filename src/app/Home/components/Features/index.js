import React from 'react';
import Button from 'common/Button';
import { MdArrowForward } from 'react-icons/md';
import cn from 'classnames';
import {
  backgroundImage,
  container,
  title,
  content,
  column,
  subtitle,
  text,
  actionButtons,
  actionButton,
  cards,
  card,
  cardTitle,
  cardText,
} from './Features.module.scss';
import FeaturesBackground from './featuresBackground.png';

const CARDS = Array.from(Array(3));

export default function Features() {
  return (
    <div className={container}>
      <div
        className={backgroundImage}
        style={{ backgroundImage: `url(${FeaturesBackground})` }}
      />
      <h1 className={title}>Funcionalidades e Benefícios do Terrafinder</h1>

      <div className={content}>
        <div className={column}>
          <h2 className={subtitle}>
            Conheça e aprenda a utilizar o Terrafinder
          </h2>

          <p className={text}>
            Você que quer vender seu imóvel rural, encontrou o lugar ideal para
            anuncia-lo. É simples, rápido e eficiente.
          </p>

          <p className={text}>
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Nesciunt
            deserunt quisquam accusantium accusamus aliquid laborum nam quasi
            earum quaerat voluptate ratione cumque tempora minima, nihil sequi.
            Culpa, ut ad? At?
          </p>

          <div className={actionButtons}>
            <Button className={actionButton}>
              <p>Quero anunciar meu imóvel</p>

              <MdArrowForward />
            </Button>

            <Button className={cn('bg-green', actionButton)}>
              <p>Quero achar uma propriedade</p>
              <MdArrowForward />
            </Button>
          </div>
        </div>

        <div className={column}>
          {CARDS.map((_, i) => (
            <div className={cards} key={String(i)}>
              <div className={card}>
                <p className={cardTitle}>
                  Pague apenas pelo tempo do seu anúncio
                </p>

                <p className={cardText}>
                  Lorem ipsum dolor sit amet consectetur adipisicing elit. Ea
                  eius eum praesentium quos delectus vero commodi ipsum
                  molestiae quod vel nostrum rerum alias minima explicabo
                  dignissimos, iste tempora nulla perferendis.
                </p>
              </div>
            </div>
          ))}
        </div>
      </div>
    </div>
  );
}
