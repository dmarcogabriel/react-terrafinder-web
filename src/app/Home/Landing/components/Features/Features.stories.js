import React from 'react';
import { Features } from './Features';

export default {
  title: 'Home > Landing > Features',
  component: Features,
  argTypes: {},
};

const Component = (args) => <Features {...args} />;

export const FeaturesComponent = Component.bind();
