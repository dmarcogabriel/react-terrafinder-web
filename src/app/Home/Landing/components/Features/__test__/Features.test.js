import React from 'react';
import ReactDOM from 'react-dom';
import { fireEvent, render } from '@testing-library/react';
import { withTheme, renderWithTheme } from 'helpers/test-helpers/theme';
import Features from '..';

let pushResponse;
const mockPush = jest.fn((url) => {
  pushResponse = url;
});
jest.mock('react-router-dom', () => ({
  useHistory: () => ({
    push: mockPush,
  }),
}));

describe('<Property />', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(withTheme(<Features />), div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('should to to create property page', () => {
    const { getByTestId } = render(withTheme(<Features />));

    const createPropertyButton = getByTestId('createProperty');
    fireEvent.click(createPropertyButton);

    expect(mockPush).toHaveBeenCalled();
    expect(pushResponse).toEqual('/create/property?step=1');
  });

  it('should to to property list page', () => {
    const { getByTestId } = render(withTheme(<Features />));

    const findPropertyButton = getByTestId('findProperty');
    fireEvent.click(findPropertyButton);

    expect(mockPush).toHaveBeenCalled();
    expect(pushResponse).toEqual('home/search-property');
  });

  it('matches snapshot', () => {
    const tree = renderWithTheme(<Features />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
