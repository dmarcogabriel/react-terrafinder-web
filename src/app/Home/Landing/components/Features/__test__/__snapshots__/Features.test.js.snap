// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Property /> matches snapshot 1`] = `
<div>
  <div />
  <h1>
    Funcionalidades e Benefícios do Terrafinder
  </h1>
  <div>
    <div>
      <h2>
        Conheça e aprenda a utilizar o Terrafinder
      </h2>
      <p>
        Você que quer vender seu imóvel rural, encontrou o lugar ideal para anuncia-lo. É simples, rápido e eficiente.
      </p>
      <p>
        Ou caso tenha interesse em comprar algum imóvel rural em determinado lugar, você achou o lugar correto! Escolha uma das opções abaixo para entender e aprender a criar ou procurar sua propriedade rural.
      </p>
      <div>
        <button
          className="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButtonBase-root css-sghohy-MuiButtonBase-root-MuiButton-root"
          data-testid="createProperty"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onContextMenu={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          Quero anunciar meu imóvel
          <span
            className="MuiButton-endIcon MuiButton-iconSizeMedium css-9tj150-MuiButton-endIcon"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="ArrowForwardIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="m12 4-1.41 1.41L16.17 11H4v2h12.17l-5.58 5.59L12 20l8-8z"
              />
            </svg>
          </span>
        </button>
        <button
          className="MuiButton-root MuiButton-contained MuiButton-containedSuccess MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButtonBase-root css-ke5b6m-MuiButtonBase-root-MuiButton-root"
          data-testid="findProperty"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onContextMenu={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          Quero achar uma propriedade
          <span
            className="MuiButton-endIcon MuiButton-iconSizeMedium css-9tj150-MuiButton-endIcon"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="ArrowForwardIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="m12 4-1.41 1.41L16.17 11H4v2h12.17l-5.58 5.59L12 20l8-8z"
              />
            </svg>
          </span>
        </button>
      </div>
    </div>
    <div
      className="MuiBox-root css-33ooz8"
    >
      <div
        className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-9xljzi-MuiPaper-root-MuiCard-root"
      >
        <div
          className="MuiBox-root css-pmxlne"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1kicsxo-MuiSvgIcon-root"
            data-testid="AccessTimeIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M11.99 2C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zM12 20c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
            />
            <path
              d="M12.5 7H11v6l5.25 3.15.75-1.23-4.5-2.67z"
            />
          </svg>
          <p
            className="MuiTypography-root MuiTypography-body1 css-ndaqpv-MuiTypography-root"
          >
            Pague apenas pelo tempo do seu anúncio
          </p>
        </div>
        <p
          className="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
        >
          Para anunciar na plataforma, selecione o plano de anúncio que melhor se encaixa para você e pague apenas pelo tempo em que ele ficar publicado.
        </p>
      </div>
      <div
        className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-i8o2f6-MuiPaper-root-MuiCard-root"
      >
        <div
          className="MuiBox-root css-pmxlne"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-8p38gn-MuiSvgIcon-root"
            data-testid="TrackChangesIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="m19.07 4.93-1.41 1.41C19.1 7.79 20 9.79 20 12c0 4.42-3.58 8-8 8s-8-3.58-8-8c0-4.08 3.05-7.44 7-7.93v2.02C8.16 6.57 6 9.03 6 12c0 3.31 2.69 6 6 6s6-2.69 6-6c0-1.66-.67-3.16-1.76-4.24l-1.41 1.41C15.55 9.9 16 10.9 16 12c0 2.21-1.79 4-4 4s-4-1.79-4-4c0-1.86 1.28-3.41 3-3.86v2.14c-.6.35-1 .98-1 1.72 0 1.1.9 2 2 2s2-.9 2-2c0-.74-.4-1.38-1-1.72V2h-1C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10c0-2.76-1.12-5.26-2.93-7.07z"
            />
          </svg>
          <p
            className="MuiTypography-root MuiTypography-body1 css-ndaqpv-MuiTypography-root"
          >
            Quero um anúncio profissional
          </p>
        </div>
        <p
          className="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
        >
          Nós oferecemos um serviço que profissionaliza o seu anúncio. Temos uma equipe que pode ir até seu imóvel, fotografa-lo e gravar vídeos sensacionais com drones. Impossível não vender assim!
        </p>
      </div>
      <div
        className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-i8o2f6-MuiPaper-root-MuiCard-root"
      >
        <div
          className="MuiBox-root css-pmxlne"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-8p38gn-MuiSvgIcon-root"
            data-testid="SupportAgentIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M21 12.22C21 6.73 16.74 3 12 3c-4.69 0-9 3.65-9 9.28-.6.34-1 .98-1 1.72v2c0 1.1.9 2 2 2h1v-6.1c0-3.87 3.13-7 7-7s7 3.13 7 7V19h-8v2h8c1.1 0 2-.9 2-2v-1.22c.59-.31 1-.92 1-1.64v-2.3c0-.7-.41-1.31-1-1.62z"
            />
            <circle
              cx="9"
              cy="13"
              r="1"
            />
            <circle
              cx="15"
              cy="13"
              r="1"
            />
            <path
              d="M18 11.03C17.52 8.18 15.04 6 12.05 6c-3.03 0-6.29 2.51-6.03 6.45 2.47-1.01 4.33-3.21 4.86-5.89 1.31 2.63 4 4.44 7.12 4.47z"
            />
          </svg>
          <p
            className="MuiTypography-root MuiTypography-body1 css-ndaqpv-MuiTypography-root"
          >
            Precisa de ajuda?
          </p>
        </div>
        <p
          className="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
        >
          Entre em contato através dos nossos canais de atendimento.

WhatsApp: (14)99999-8888
        </p>
      </div>
    </div>
  </div>
</div>
`;
